packages/firestore/exp/test/shim.ts

```diff
@@ -99,6 +99,10 @@ export class FirebaseFirestore
     initializeFirestore(this.app._delegate, settings);
   }
 
+  useEmulator(host: string, port: number) {
+    this.settings({ host: `${host}:${port}`, ssl: false, merge: true });
```

Comment:
This feels a little sketch but I think it's fine for the shim?

Reply:
Works for me. I do think we can add it to firestore-exp with not a whole lot of trouble, but we can do this a  follow up for sure:

```
function useEmulator(db: Firestore, host: string, port: number): 
```

The only tricky part is that this will be the only API that modifies the settings after startup.

Reply:
Ok sgtm.  I think it will have to be called `useFirestoreEmulator` so that we don't conflict between vNext packages.

Should I file a bug to track this TODO?

Reply:
Yes a bug would help (and `useFirestoreEmulator` make sense)

Reply:
http://b/170936261